README.md

```diff
@@ -1,3 +1,21 @@
 # GOV.UK Frontend Jinja Macros
 
 This repository contains [Jinja](https://jinja.palletsprojects.com/) macro ports of the [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend). This is currently up to date with [release v3.7.0](https://github.com/alphagov/govuk-frontend/releases/tag/v3.7.0).
+
+## How to use
+
+[Here](https://github.com/matthew-shaw/govuk-frontend-jinja-example) is a simple example Flask app that demonstrates how to import and use the GOV.UK Jinja macros.
+
+After running `pip install govuk-frontend-jinja`, ensure that you tell Jinja where to load the templates from using the `PackageLoader` as follows:
```

Comment:
Assuming some people won't look at the example, it might be worth a small note in here pointing out that people will need to import their templates from the new namespaces. I.e. `render_template('app/index.html')` and similar for imports inside the templates.

Alternatively, is it worth trying something like this:

```
loader=ChoiceLoader([
    PackageLoader('app'),
    PrefixLoader({
        'govuk_frontend_jinja': PackageLoader('govuk_frontend_jinja')
    })
])
```

I haven't tried this code, but I _think_ it should allow people to use templates local to their app without an namespace prefix. It would be more "normal".